crates/ln-dlc-node/src/node.rs

```diff
@@ -818,9 +850,14 @@ pub(crate) fn coordinator_config() -> UserConfig {
             // between the counterparties
             max_minimum_depth: 1,
             trust_own_funding_0conf: true,
-            // Enforces incoming channels to the coordinator to be public! We
-            // only want to open private channels to our 10101 app.
-            force_announced_channel_preference: true,
+
+            // TODO: The config `force_announced_channel_preference` has been temporarily disabled
+            // for testing purposes, as otherwise the app is not able to open a channel to the
+            // coordinator. Uncomment belows comment once `force_announced_channel_preference` is
+            // set back to `true`
+            // // Enforces incoming channels to the coordinator to be public! We
+            // // only want to open private channels to our 10101 app.
+            force_announced_channel_preference: false,
```

Comment:
🙃 I would prefer to instead use `Node::new` in the `coordinator` and pass in a mutated version of the `coordinator_config` with the `force_announced_channel_preference` set to false.

Reply:
agreed.